### PR TITLE
Granspencer09 patch 26

### PR DESCRIPTION
**Pull request by msh100**
Merge gh-pages into #117

### CODE REVIEW

File: enterprise/cluster-alerting.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ title: "Cluster Alerting Preferences"
 keywords: lighthouse
 sidebar: home_sidebar
 redirect_from: "/Lighthouse-Manage-Clusters.html"
+meta-description: "To setup your Lighthouse cluster alert settings, use these four simple steps. See for yourself and get customizing!"
```
Comment (msh100) [inline enterprise/cluster-alerting.md]: Why highlighted in yellow on spreadsheet?

File: knowledgebase/selinux.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ title: "PX with SELinux"
 keywords: portworx, troubleshooting, logs, issue
 sidebar: home_sidebar
 redirect_from: "/selinux.html"
+meta-description: "Are you getting a “No such file or directory” message when you use SELinux? Portworx has a solution to resolve the issue."
```
Comment (msh100) [inline knowledgebase/selinux.md]: Confirm these quotes are okay, suspect may be problematic

Comment (msh100) [inline knowledgebase/selinux.md]: also yellow on spreadsheet

File: knowledgebase/sles-12-sp1.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ title: "Running PX on SuSe Enterprise Linux 12SP1"
 keywords: portworx, px-developer, px-enterprise, install, configure, SLES 12, SP1
 sidebar: home_sidebar
 redirect_from: "/run-px-on-sles12-sp1.html"
+meta-description: "Follow our step-by-step instructions to running Portworx on SuSe Enterprise Linux 12SP1"
```
Comment (msh100) [inline knowledgebase/sles-12-sp1.md]: SuSE has an uppercase E, also does not end in punctuation.

Comment (msh100) [inline knowledgebase/sles-12-sp1.md]: also highlighted in yellow

File: scheduler/mesosphere-dcos/volume-scaling.md [hunk 4]
```diff
@@ -3,6 +3,7 @@ layout: page
 title: "Volume Scaling with DCOS"
 keywords: portworx, jenkins
 sidebar: home_sidebar
+meta-description: "Learn how to implement and use volume scaling with DCOS. Try it today!"
```
Comment (msh100) [inline scheduler/mesosphere-dcos/volume-scaling.md]: Highlighted in yellow on spreadsheet

File: cloud/azure/azure.md [hunk 5]
```diff
@@ -4,6 +4,7 @@ title: "Portworx on Azure"
 keywords: portworx, azure, PaaS, IaaS, docker, converged, cloud
 sidebar: home_sidebar
 redirect_from: "/portworx-on-azure.html"
+meta-description: "Follow these four steps to run Portworx on Azure. Portworx outlines the process to do so!"
```
Comment (ferrantim) [inline cloud/azure/azure.md]: Just say "Follow these four steps to run Portworx on Azure. Try it today!"

File: cloud/terraform.md [hunk 6]
```diff
@@ -4,6 +4,7 @@ title: "Terraform"
 keywords: terraform, automation, terraporx, cloud
 sidebar: home_sidebar
 redirect_from: "/terraporx.html"
+meta-description: "Portworx has open-sourced the Terraporx repository, containing Terraform scripts for a number of cloud providers. Check it out!"
```
Comment (ferrantim) [inline cloud/terraform.md]: you can ignore that. It was to keep track of where we were in the spreadsheet. This is fine

File: control/host.md [hunk 7]
```diff
@@ -4,6 +4,7 @@ title: "CLI Reference–Host"
 keywords: portworx, pxctl, command-line tool, cli, reference
 sidebar: home_sidebar
 redirect_from: "/cli-reference.html"
+meta-description: "This guide shows you how to use the PXCL CLI to perform host operations. Try today"
```
Comment (ferrantim) [inline control/host.md]: add a period.

File: enterprise/cluster-alerting.md (hunk 1 above)
Comment (ferrantim) [inline enterprise/cluster-alerting.md]: Not sure, this is fine though

File: knowledgebase/selinux.md (hunk 2 above)
Comment (ferrantim) [inline knowledgebase/selinux.md]: You can remove the quotes.

File: knowledgebase/sles-12-sp1.md (hunk 3 above)
Comment (ferrantim) [inline knowledgebase/sles-12-sp1.md]: you can update the spelling and add a period

File: scheduler/mesosphere-dcos/volume-scaling.md (hunk 4 above)
Comment (ferrantim) [inline scheduler/mesosphere-dcos/volume-scaling.md]: you can go ahead with this

**Pull request merged**